### DISCUSSION TITLE
Using board without decree feature

### DISCUSSION THREAD

**Discussion by Developer2704** [ejektaflex/Bountiful#230] [Mod Ideas]
### What happened?

Can you make it so that the bounty board can be used without a decree so it will just give random bounties?

### Mod Version

6.00 1.19.4 forge

### Mod Loader

Forge

### Mod Loader Version

45.0.43

**Comment by ejektaflex** [comment]
This is a great idea! If the config is enabled, we can just tell it to ignore decrees in the decree slots and create bounties from all loadable decrees instead.
